maintenance/rebuildLocalisationCache.php

```diff
@@ -151,6 +158,28 @@ function ( $item ) use ( $firstCodes ) {
 	}
 
 	/**
+	 * Helper function to include all localisation files to localisation cache rebuild process.
+	 * It looks for i18n and alias files from whole code directory and registers them in global array.
+	 */
+	private function includeAllMessagesFiles() {
+		global $wgExtensionMessagesFiles;
+
+		$directory = new RecursiveDirectoryIterator('/usr/wikia/source/app/');
+		$iterator = new RecursiveIteratorIterator($directory);
+		$files = new RegexIterator($iterator, self::LOCALISATION_FILE_REGEX, RecursiveRegexIterator::GET_MATCH);
+
+		foreach ($files as $filepath) {
+			if (!isset($filepath[0])) continue;
```

Comment:
It should not happen, as if file exists there will always be `[0]` that contains full path `and `[1]` the substring that matches to regex. Changed in : 725ee7e